### ISSUE TITLE
Investigate ability to remove requirement of registry deployment

### ISSUE THREAD

**Issue by dymurray** [openshift/oadp-operator#178]
Talking with Shawn, we want to explore the ability to remove the registry entirely from our architecture. It should be possible for the openshift plugin to simply export the container FS and tar it up and store it in the s3 bucket. This would remove the registry entirely.

**Comment by sseago** [comment]
That would be a big help. When I initially wrote the plugin code, I tried to find a way to do this with the existing containers/image and/or skopeo APIs, but if I'm remembering correctly, I was able to find a way to do the transfer in one direction but not the other way. I may have missed something, or we may need a different tool that can handle both registry-to-tarball and tarball-to-registry in a way that skopeo/containers/image could not.
